libs/stylelint-config/index.js

```diff
@@ -118,7 +118,9 @@ module.exports = {
     "declaration-block-no-duplicate-custom-properties": true,
     "declaration-block-no-duplicate-properties": true,
     "declaration-block-no-shorthand-property-overrides": true,
-    "declaration-block-no-redundant-longhand-properties": true,
+    "declaration-block-no-redundant-longhand-properties": [
+      true, { "ignoreShorthands": [] }
```

Comment:
The ignoreLonghands: [] option was introduced in Stylelint version 16.4.0. We might want to consider whether there are any scenarios where certain longhand properties should be ignored
https://stylelint.io/user-guide/rules/declaration-block-no-redundant-longhand-properties/

Reply:
Depending on the browsers you are targetting that would be `"text-decoration-thickness"` and/or `"background-size", "background-origin", "background-clip"`.

Reply:
Depending on the browsers you are targetting that would be `"text-decoration-thickness"` and/or `"background-size", "background-origin", "background-clip"`.

Reply:
Hi @Mouvedia! Many thanks for all the efforts in making Stylelint great library and helping us here as well!
Keep up the good work!

As for our support, we do not care of IE or older versions of browsers but Safari happens to be the most fragile lately. I'd go with empty array unless you had experience with some of the mentioned ones in combination with Safari?

Reply:
👋🏻

> I'd go with empty array unless you had experience with some of the mentioned ones in combination with Safari?

check https://caniuse.com/?search=text-decoration-thickness and https://caniuse.com/background-img-opts

Reply:
Still sounds like an empty array :) I guess there's only one way to find out...

Reply:
You might want to add `"transition-behavior"` retroactively.
i.e. it's not supported by the rule yet